app/interfaces/api/entities/cclf/base_claim.rb

```diff
@@ -32,6 +32,11 @@ def actual_trial_length_or_one
           object.actual_trial_length.or_one
         end
 
+        # case type adapter requires access to claim object
+        def case_type
```

Comment:
Following our offline discussion is probably worth leaving a comment here explaining the reason behind this.

Reply:
did that in a refactor last night.